addons/dataverse/models.py

```diff
@@ -125,6 +125,24 @@ def nodelogger(self):
             auth=auth
         )
 
+    def get_folders(self, path=None, folder_id=None):
+        """Get list of datasets from provided Dataverse alias"""
+
+        connection = client.connect_from_settings(self)
+        dataverse = client.get_dataverse(connection, self.dataverse_alias)
+
+        datasets = client.get_datasets(dataverse)
```

Comment:
Regarding my work on refactoring citations for API v2 (https://github.com/CenterForOpenScience/osf.io/pull/8230) Zotero also has two (and sometimes more) levels,  You first pick your library, which can be a group library or your personal library, which I think can be compared to your dataverse.  Then from that library, you select your collection, which can be equated to a dataset. 

How I structured the API folders endpoint for Zotero, was top-level was the library.  And children of that library were collections, and then children of those collections were more collections (since you can have collections within collections).   My API can go multiple levels.  

Is this what is wanted for Dataverse? Top level get_folders returns dataverses, and then the children link returns datasets.  (Granted, my PR is in Final Review right now, and I may have misunderstood what was wanted here, so check with @sloria.) 

Edit: Perhaps Zotero is different, in that library selection and folder selection are two explicit steps, saved separately, and dataverse it looks like a 1-step thing.  For Zotero, you can have a library saved, but no collection.

Reply:
I talked to @sloria about this and it's going to require multiple steps. I think we can still get this step out though.